### DISCUSSION TITLE
How to migrate sqlite database from one installation to other

### DISCUSSION THREAD

**Discussion by patelbhavinoutlook** [nocodb/nocodb#3942] [Q&A]
Hello,

Can anyone help me to understand how can i migrate or backup sqlite database form one nocodb docker to other..

**Comment by dstala** [comment]
Assuming your question is about NocoDB meta DB. 
SQLite file will be within the same folder where the volume is mounted. You can copy that file for backup or re-use it in a different docker instance.
